paasta_tools/instance/kubernetes.py

```diff
@@ -446,6 +450,19 @@ def kubernetes_status_v2(
     if kube_client is None:
         return status
 
+    if (
```

Comment:
Do you think it would be better to only return this in the API if verbose > 1, rather than always returning it in the API and then limiting the output in the CLI?

Reply:
I've gated this with `verbose > 1` now